app/tests/spec/views/base.js

```diff
@@ -40,7 +38,7 @@ function (chai, $, sinon, BaseView, p, Translator, EphemeralMessages, Metrics,
     var translator;
     var metrics;
     var fxaClient;
-    var relier;
+    var relier; //eslint-disable-line no-unused-vars
```

Comment:
Done, and I found why I couldn't do this yesterday. So if we remove the relier dependency, we need to remove the formal parameter Relier as well for the tests to pass :D I missed that yesterday, which is why the "firefox platform" error was thrown :D